.travis.yml

```diff
@@ -18,12 +26,14 @@ before_install:
     - popd
     - cmake --version
 install:
-    - sudo pip install -r requirements.txt -r requirements-dev.txt -r plugins/geospatial/requirements.txt -r plugins/metadata_extractor/requirements.txt
-    - sudo pip install -r plugins/celery_jobs/requirements.txt -r plugins/hdfs_assetstore/requirements.txt
+    - mkdir -p "${HOME}/.local/bin"
+    - export PATH="${HOME}/.local/bin:${PATH}"
+    - pip install --user -r requirements.txt -r requirements-dev.txt -r plugins/geospatial/requirements.txt -r plugins/metadata_extractor/requirements.txt
+    - pip install --user -r plugins/celery_jobs/requirements.txt -r plugins/hdfs_assetstore/requirements.txt
     # We have to upgrade six or requiring moto can cause other modules to fail
-    - sudo pip install -U six
-    - sudo python -c "import bcrypt"
-    - npm install -g grunt grunt-cli
+    - pip install --user -U six
```

Comment:
I don't think the `--user` is strictly necessary because travis builds are always done in a userspace virtualenv.  Getting rid of the sudo probably make the initial `import bcrypt` below unnecessary as well.